.travis.yml

```diff
@@ -20,6 +20,9 @@ before_script:
   - wget https://artifacts.crowdin.com/repo/deb/crowdin.deb -O crowdin.deb
   - sudo dpkg -i crowdin.deb
 
+script:
+   - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then yarn translate:upload'
```

Comment:
not sure if it's in the right section though: https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions

Reply:
/cc @Yserz